src/UI/Internal/Dialog.elm

```diff
@@ -59,6 +60,12 @@ desktopDialogView cfg { title, body, close } =
             (desktopHeaderRow cfg close title)
         , Element.paddingEach
             { top = 8, left = 32, right = 32, bottom = 32 }
+        , Border.roundEach
+            { topLeft = 0
+            , topRight = 0
+            , bottomLeft = 6
+            , bottomRight = 6
+            }
```

Comment:
(Not anything to be changed here rn, just a thought while skimming through) 

Maybe we should start complementing paack-ui with more base rules of the design system other than color and typography. Spacing and border radius are two that are missing. I would love to have something like:

```elm
[ UI.Space.inset -- perhaps supplying a t-shirt size S, M, G, XG, etc
, UI.Radius.round -- and other options like square, circle, etc
-- more stuff I can't think of?
]
```

here's a great read about dealing with whitespace on design systems: https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62